lib/easy_broker/version.rb

```diff
@@ -1,3 +1,3 @@
 module EasyBroker
-  VERSION = "0.1.4"
+  VERSION = "2.0.0"
```

Comment:
```suggestion
  VERSION = "0.2.0"
```

Reply:
Brincaste versión 1.0.
```suggestion
  VERSION = "1.0.0"
```

Reply:
Ups, all tests are ok 👍